CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3539,6 +3562,10 @@
 					8064D9971C4D27E9005A8B4C = {
 						DevelopmentTeam = 5Z97G9NZQ6;
 					};
+					C25664D123571D330088513E = {
+						CreatedOnToolsVersion = 11.1;
```

Comment:
Do you use Xcode 11.1? It doesn't matter unless higher versions cause any issues but I wonder if you are using Xcode 11.1 to generate the lib for App Center.

Reply:
I tested on the last (11.1) and 10.1.
CI uses 10.1 for lib generation.